astropy/wcs/docstrings.py

```diff
@@ -1024,7 +1024,7 @@ def RA_DEC_ORDER(indent=0):
 """
 
 DistortionLookupTable = """
-DistortionLookupTable(*table*, *crpix*, *crval*, *cdelt*)
+DistortionLookupTable(table, crpix, crval, cdelt)
```

Comment:
No need to remove the formatting here?

Reply:
I'm happy to leave it, it just looked very non-standard---it comes through in the [HTML docs](https://docs.astropy.org/en/stable/api/astropy.wcs.DistortionLookupTable.html#astropy.wcs.DistortionLookupTable), where I've never seen asterisks like that before.

Reply:
Oh, I thought it was to make it *italic* but maybe that isn't the case. OK, let's leave your removal as-is and see what `wcs` maintainers say. Thanks!